integration-cli/docker_cli_logs_bench_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/go-check/check"
+)
+
+func (s *DockerSuite) BenchmarkLogsCLIRotateFollow(c *check.C) {
```

Comment:
I added this a benchmark b/c I figured it's not really helpful to have a test that we know will always take a long time.. and it really kind of is a benchmark... like "can the log code handle the load".